### PR TITLE
Create loader-component.md

### PR DESCRIPTION
**Pull request by thekiba**

### CODE REVIEW

File: snippets/loader-component.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# Content
+You can create own helper component and use it instead of `*ngIf`.
+
+```typescript
+@Component({
+  selector: 'loader',
+  template: `
+    <ng-content *ngIf="!loading else showLoader"></ng-content>
+    <ng-template #showLoader>
+      Come away with me to the Anywhere City!
```
Comment (NothingEverHappens) [inline snippets/loader-component.md]: LOL what's this?

File: snippets/loader-component.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  `
+})
+class LoaderComponent {
+  @Input() loading: boolean;
+}
+```
+
+For usage example:
+```html
+<loader [loading]="loading">🦊 🦄 🐉</loader>
```
Comment (NothingEverHappens) [inline snippets/loader-component.md]: Shouldn't this be a * directive?

Comment (thekiba) [reply snippets/loader-component.md]: It's should be component. But it can be a * directive too.

**Review by alex-bu-93 — APPROVED** [review]

File: snippets/loader-component.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+---
+
+# Content
+You can create own helper component and use it instead of `*ngIf`.
+
+```typescript
+@Component({
+  selector: 'loader',
+  template: `
+    <ng-content *ngIf="!loading else showLoader"></ng-content>
```
Comment (fetis) [inline snippets/loader-component.md:19]: missed `;`?
```suggestion
    <ng-content *ngIf="!loading; else showLoader"></ng-content>
```

File: snippets/loader-component.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    <ng-template #showLoader>🕚 Wait 10 seconds!</ng-template>
+  `
+})
+class LoaderComponent {
+  @Input() loading: boolean;
```
Comment (fetis) [inline snippets/loader-component.md:24]: I would change it to `isLoading` everywhere

File: snippets/loader-component.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+---
+title: Loader Component
+level: beginner
+author: thekiba
+tags:
+  - tips
```
Comment (fetis) [inline snippets/loader-component.md:6]: tips and good-to-know are not informative imo. I would add `loader`, `ngIf`

**Review by fetis — CHANGES_REQUESTED** [review]

File: snippets/loader-component.md (hunk 3 above)
Comment (thekiba) [reply snippets/loader-component.md:19]: Without `;`

Comment (fetis) [reply snippets/loader-component.md:19]: official doc suggests semicolon
https://angular.io/api/common/NgIf

Comment (thekiba) [reply snippets/loader-component.md:19]: Without `;`

**Review by fetis — APPROVED** [review]

File: snippets/loader-component.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  `
+})
+class LoaderComponent {
+  @Input() loading: boolean;
+}
+```
+
+For usage example:
+```html
+<loader [loading]="isLoading">🦊 🦄 🐉</loader>
```
Comment (kirjs) [inline snippets/loader-component.md:30]: Maybe use structural directive?
<loader [loading]="isLoading">
   <destroy-the-world></destroy-the-world>
</loader>

Comment (kirjs) [reply snippets/loader-component.md:30]: Or add a note

**Pull request merged**